.github/workflows/release.yml

```diff
@@ -90,8 +88,6 @@ jobs:
         shell: powershell
         run: |
           New-Item -Path archive -ItemType Directory
-          Copy-Item target/release/dora-coordinator.exe -Destination archive
-          Copy-Item target/release/dora-daemon.exe -Destination archive
           Copy-Item target/release/dora.exe -Destination archive/dora.exe
           Compress-Archive -Path archive\* -DestinationPath archive.zip
```

Comment:
As we only distribute a single binary as artifact, we don't need to zip it. Or is there any advantage in zipping it?